fe/fe-core/src/main/java/com/starrocks/connector/ColumnTypeConverter.java

```diff
@@ -175,6 +176,36 @@ public static Type fromHudiType(Schema avroSchema) {
                     primitiveType = PrimitiveType.VARCHAR;
                     break;
                 }
+            case RECORD:
+                // Struct type
+                List<Schema.Field> fields = avroSchema.getFields();
+                Preconditions.checkArgument(fields.size() > 0);
+                ArrayList<StructField> structFields = new ArrayList<>(fields.size());
+                for (Schema.Field field : fields) {
+                    String fieldName = field.name();
+                    Type fieldType = fromHudiType(field.schema());
+                    if (fieldType.isUnknown()) {
+                        isConvertedFailed = true;
+                        break;
```

Comment:
One of the subfield of Struct is not recognized by SR, the others still able to be query right? Seems we don't have to terminate the entire convert procedure.